homeassistant/components/gandi_livedns/translations/fr.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Translations are managed via Lokalise initially only the source translations and `en` should be included, please remove additional languages.

Reply:
I have removed fr language